src/content/en/tools/lighthouse/cache-contains-start_url.md

```diff
@@ -0,0 +1,55 @@
+project_path: /web/_project.yaml
+book_path: /web/tools/_book.yaml
+description: Reference documentation for the "Cache contains start_url from manifest" Lighthouse audit.
+
+{# wf_updated_on: 2016-09-15 #}
+{# wf_published_on: 2016-09-15 #}
+
+# Cache contains start_url from manifest  {: .page-title }
```

Comment:
I wrote these titles to match how they are presented in Lighthouse reports, but I guess it's not a big deal to change them so they match up with the rest of /web.